transition-scripts/test-dns.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+usage() {
+    cat <<EOF
+Test that a url points at a specific DNS
+
+Usages:
+    $0 url cluster
+
+Available clusters:
+    - gold
+    - golddr
+
+Examples:
+    $ $0 dev.sandbox.loginproxy.con gold
```

Comment:
was it supposed to be `dev.sandbox.loginproxy.com`??

Reply:
fixed